.github/ISSUE_TEMPLATE/general_report.yml

```diff
@@ -1,5 +1,5 @@
 name: 📝 General Issue
-description: Report a bug, request documentation updates, or ask a general question
+description: Report a bug or request documentation updates
```

Comment:
Why no longer ask a general question?

Reply:
Questions fit better in https://github.com/open-edge-platform/geti/discussions

Reply:
Should we add a section to point users to the discussions? Many repos haven't adopted the discussions tab yet, so it might not be immediately clear to everyone.

Reply:
It is mentioned here: https://github.com/open-edge-platform/geti/blame/f6da0edafe8d90ae5301a610537528a76b0772b4/README.md#L163

Otherwise they can open a "Blank issue"

![image](https://github.com/user-attachments/assets/136424cc-344c-47c1-a0b8-02c42a5c4d2e)